types/index.d.ts

```diff
@@ -47,4 +47,4 @@ declare class HCaptcha extends React.Component<HCaptchaProps, HCaptchaState> {
   execute(opts?: { async: boolean }): Promise<ExecuteResponse> | void;
 }
 
-export = HCaptcha;
+export default HCaptcha;
```

Comment:
Are there typescript tests we can add to prevent regression?

Reply:
in fact, let me check once again without this change

Reply:
This was resolved without a reply. 

It would be nice to avoid force pushing during review as it is hard to keep track of what was changed since the last review  @zoryana94